README.md

```diff
@@ -1,3 +1,56 @@
 # interactive-plugins
 
-A project generated from [aunty](https://github.com/abcnews/aunty)'s `svelte` project template.
+This project provides a set of plugins that editors can include in articles to let readers control their experience.
+
+## Plugins
+
+### AppAudioVisualToggle
+
+Allows readers to control autoplay sound in videos embedded within articles.
+
+#### Usage
+
+1. Embed the Javascript document in your article (“[odyssey audio visual plugin]” - Content ID `#101718598`)
```

Comment:
Yes good point. It does replace it. I added a note here, but not super prominently because, well, I'm reminded of this toot 😆 https://bne.social/@lynnesbian@fedi.lynnesbian.space/112742421931778621